src/wags_tails/drugbank.py

```diff
@@ -0,0 +1,100 @@
+"""Provide source fetching for DrugBank."""
+import logging
+from pathlib import Path
+from typing import Tuple
+
+import requests
+
+from .base_source import DataSource, RemoteDataError
+from .utils.downloads import download_http, handle_zip
+from .utils.versioning import parse_file_version
+
+_logger = logging.getLogger(__name__)
+
+
+class DrugBankData(DataSource):
+    """Provide access to DrugBank database."""
+
+    _src_name = "drugbank"
+    _filetype = "csv"
+
+    @staticmethod
+    def _get_latest_version() -> Tuple[str, str]:
+        """Retrieve latest version value
+
+        :return: latest release value and base download URL
+        :raise RemoteDataError: if unable to parse version number from releases API
+        """
+        releases_url = "https://go.drugbank.com/releases.json"
+        r = requests.get(releases_url)
+        r.raise_for_status()
+        try:
+            latest = r.json()[0]
+            return latest["version"], latest["url"]
+        except (KeyError, IndexError):
+            raise RemoteDataError(
+                "Unable to parse latest DrugBank version number from releases API endpoint"
+            )
+
+    def _get_latest_local_file(self, glob: str) -> Tuple[Path, str]:
+        """Get most recent locally-available file. DrugBank uses versioning that isn't
+        easily sortable by default so we have to use some extra magic.
+
+        :param glob: file pattern to match against
+        :return: Path to most recent file, and its version
+        :raise FileNotFoundError: if no local data is available
+        """
+        _logger.debug(f"Getting local match against pattern {glob}...")
+        file_version_pairs = []
+        for file in self.data_dir.glob(glob):
+            version = parse_file_version(file, r"drugbank_([\d\.]+).csv")
+            formatted_version = [int(digits) for digits in version.split(".")]
+            file_version_pairs.append((file, version, formatted_version))
+        files = list(sorted(file_version_pairs, key=lambda p: p[2]))
+        if len(files) < 1:
```

Comment:
Unless I'm missing something, I think we can just do this

```suggestion
        if not files:
```